examples/hosts/app-integration/external-data/src/view/taskListView.tsx

```diff
@@ -8,6 +8,11 @@ import React, { useCallback, useEffect, useRef, useState } from "react";
 
 import type { ExternalSnapshotTask, ITask, ITaskList } from "../model-interface";
 
+/**
+ * {@link TaskRow} input props.
+ * leader: the clientId of the current leader in the container, could be same as clientID. No leader if undefined.
+ * clientID: the current user's clientId. If undefined, client is not connected to container
+ */
 interface ITaskRowProps {
 	readonly leader: string | undefined;
 	readonly clientID: string | undefined;
```

Comment:
It's best to put docs inline over each property in the interface so they show up with IntelliSense when people hover over the property names in the code. Also includes a few small corrections:
```suggestion
interface ITaskRowProps {
	/*
	 * The clientId of the current leader, or undefined if no client has claimed leadership.
	 */
	readonly leader: string | undefined;
	/*
	 * The current client's id. If undefined, the client is not connected to the service.
	 */
	readonly clientID: string | undefined;
```